docs/publications.rst

```diff
@@ -93,43 +93,77 @@ Furthermore, if you use any of the design automation algorithms, please consider
         @inproceedings{drewniok2023quicksimIEEE,
           title={{\emph{QuickSim}: Efficient \emph{and} Accurate Physical Simulation of Silicon Dangling Bond Logic}},
           author={Drewniok, Jan and Walter, Marcel and Ng, Samuel Sze Hang and Walus, Konrad and Wille, Robert},
-          booktitle={2023 IEEE 23rd International Conference on Nanotechnology (NANO)},
+          booktitle={IEEE International Conference on Nanotechnology (IEEE NANO)},
           pages={817-822},
           year={2023}
         }
 
+* :ref:`QuickExact <quickexact>` (``quickexact``):
+
+    .. code-block:: tex
+
+        @inproceedings{drewniok2024quickexact,
+          title={{The Need for Speed: Efficient Exact Simulation of Silicon Dangling Bond Logic}},
+          author={Drewniok, Jan and Walter, Marcel and Wille, Robert},
+          booktitle={Asia and South Pacific Design Automation Conference (ASP-DAC)},
+          year={2024}
+        }
+
 * :ref:`Critical temperature simulation <critical_temperature>` (``critical_temperature``):
 
     .. code-block:: tex
 
-        @inproceedings{drewniok2023temperatureIEEE,
+        @inproceedings{drewniok2023temperature,
           title={{Temperature Behavior of Silicon Dangling Bond Logic}},
           author={Drewniok, Jan and Walter, Marcel and Wille, Robert},
-          booktitle={2023 IEEE 23rd International Conference on Nanotechnology (NANO)},
+          booktitle={IEEE International Conference on Nanotechnology (IEEE NANO)},
           pages={925-930},
           year={2023}
         }
 
+* :ref:`Operational Domain computation <operational_domain_flood_fill / operational_domain_contour_tracing>` (``opdom``):
+
+    .. code-block:: tex
+
+        @inproceedings{walter2023opdom,
+          title={{Reducing the Complexity of Operational Domain Computation in Silicon Dangling Bond Logic}},
+          author={Walter, Marcel and Drewniok, Jan and Ng, Samuel Sze Hang and Walus, Konrad and Wille, Robert},
+          booktitle={International Symposium on Nanoscale Architectures (NANOARCH)},
+          year={2023}
+        }
+
 * :ref:`Mapping Cartesian to hexagonal layouts <hexagonalization>` (``hexagonalization``):
 
     .. code-block:: tex
 
         @inproceedings{hofmann2023hexagonalization,
           title={{Scalable Physical Design for Silicon Dangling Bond Logic: How a 45\textdegree~Turn Prevents the Reinvention of the Wheel}},
           author={Hofmann, Simon and Walter, Marcel and Wille, Robert},
-          booktitle={2023 IEEE 23rd International Conference on Nanotechnology (NANO)},
+          booktitle={IEEE International Conference on Nanotechnology (IEEE NANO)},
           pages={872-877},
           year={2023}
         }
 
+
+* :ref:`Post-layout optimization <post_layout_optimization>` (``post_layout_optimization``):
+
+    .. code-block:: tex
+
+        @inproceedings{hofmann2023postlayout,
+          title={{Post-Layout Optimization for Field-Coupled Nanotechnologies}},
```

Comment:
```suggestion
          title={{Post-Layout Optimization for Field-coupled Nanotechnologies}},
```

Reply:
We changed the title to "Field-coupled" to align with other publications.

Reply:
Has been changed on the CDA website already: https://github.com/cda-tum/webpage/pull/467